packages/astral/composer.json

```diff
@@ -8,7 +8,7 @@
         "nette/utils": "^3.2",
         "symfony/dependency-injection": "^5.3|^6.0",
         "symplify/smart-file-system": "^10.0",
-        "phpstan/phpstan": "^1.0",
+        "phpstan/phpstan": "~1.1.0",
```

Comment:
By convention we use ^ everywhere here. Bumping to 1.2 is ok 👍

Reply:
I know but it failed with 1.2 and I need fix for it. Then I realize how to fix it in main. I will merge it here